dev/com.ibm.ws.classloading/resources/com/ibm/ws/classloading/internal/resources/ClassLoadingServiceMessages.nlsprops

```diff
@@ -155,3 +155,7 @@ cls.classloader.config.typo.useraction=Check the server configuration for invali
 cls.classloader.config.typo2=CWWKL0083E: When using prefix + or -, [{0}] API type is not valid in this [{1}] list.  All API type must have a prefix of + or -
 cls.classloader.config.typo2.explanation=Change the invalid API type to a valid API type with + or -.
 cls.classloader.config.typo2.useraction=Check the server configuration for invalid API type to ensure that the API type are correct.
+
+cls.classloader.suggested.feature=CWWKL0084W: Unable to load class {0}, try enabling the {1} (or newer) feature in your server configuration.
+cls.classloader.suggested.feature.explanation=A class could not be loaded which is typically provided by one or more server features.
+cls.classloader.suggested.feature.useraction=A class could not be loaded which is typically provided by one or more server features. Try enabling the suggested feature in 
```

Comment:
The end of the action seems to be missing as the action ends with "Try enabling the suggested feature in ". 
Change this sentence to "Try enabling the specified feature or a newer version of the feature in your server configuration.".
Also, remove "A class could not be loaded which is typically provided by one or more server features." as this sentence is part of the explanation. We don't want to put explanation info in the action.

Reply:
> Is the feature enabled in the server.xml file?
Yes, the features are normally enabled in server.xml.  I didn't want to explicitly state server.xml in the message, in case we ever come up with alternative mechanisms for enabling features in the future, but I see there are already many other NLS messages in Liberty that directly refer to server.xml, so I will do so here.